agent/config/types.go

```diff
@@ -36,7 +36,9 @@ type Cloud struct {
 }
 
 type Opentelemetry struct {
-	Enable bool `mapstructure:"enable"`
+	Enable       bool       `mapstructure:"enable"`
+	MQTT         MQTTConfig `mapstructure:"mqtt"`
+	MetricsTopic string     `mapstructure:"metrics_topic"`
```

Comment:
this shouldn't be user configurable from the config file

Reply:
Alright, I was in doubt of that, I used this for local testing.